codegen/java-gen/src/test/scala/com/lightbend/akkasls/codegen/java/EventSourcedEntityTestKitGeneratorSuite.scala

```diff
@@ -0,0 +1,209 @@
+/*
+ * Copyright 2021 Lightbend Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.lightbend.akkasls.codegen
+package testkit
+
+class EventSourcedEntityTestKitGeneratorSuite extends munit.FunSuite {
+
+  test(
+    "it can generate an specific TestKit for the proto files " +
+    "in test/resources/testkit"
```

Comment:
I guess this is not using the file, correct? 
The test name is a bit confusing in that case. 

But that's a great idea. Maybe for another PR. Instead of declaring things like `PackageNaming`, `ModelBuilder.EntityService`, `ModelBuilder.Command`, we should read a proto file, generate code, compare with some expected output. 

And ultimately, we should compile the generated code to prove that we are generating valid java code.

Reply:
I did use the original files at the beginning but then I thought it was a bit cumbersome. The problem I encounter is that `PackageNaming`, `ModelBuilder.EntityService`, `ModelBuilder.Command` are the abstractions we have to manipulate after all.  What I found is that writing them explicitly on the test made it easier for me to map them to the protos. In any case is worth exploring it some more, sure.  

Compile the generated code sounds really good 👍

Reply:
I didn't answer your question, though. You're right I didn't use them